tests/ha/check_after_reboot.pm

```diff
@@ -5,7 +5,134 @@
 
 # Package: yast2-iscsi-client pacemaker-cli
 # Summary: Check cluster status *after* reboot
-# Maintainer: QE-SAP <qe-sap@suse.de>, Loic Devulder <ldevulder@suse.com>
+
+=head1 NAME
+
+ha/check_after_reboot.pm - Check cluster status after a reboot
+
+=head1 DESCRIPTION
+
+This module is responsible for verifying and restoring the cluster's health after a
+node has rebooted, typically as a result of a fencing operation. It performs a
+series of checks and recovery actions to ensure the cluster returns to a stable
+and operational state.
+
+The key tasks performed by this module include:
+
+=over 4
```

Comment:
Not an issue, but if I remember indentations has no effect in `=over 4`. Or is it for the documentation parsing?